doc/release-notes/6.2-release-notes.md

```diff
@@ -0,0 +1,401 @@
+This will aid in pagination by allowing the caller to know how many pages can be iterated through. The existing API (getVersionFileCounts) to return the count will still be available.
+
+### Metadata Blocks Endpoint Extended
+
+The API endpoint `/api/metadatablocks/{block_id}` has been extended to include the following fields:
+
+- **isRequired**: Whether or not this field is required
+- **displayOrder**: The display order of the field in create/edit forms
+- **typeClass**: The type class of this field ("controlledVocabulary", "compound", or "primitive")
+
+### Get File Citation As JSON
+
+It is now possible to retrieve via API the file citation as it appears on the file landing page. It is formatted in HTML and encoded in JSON.
+
+This API is not for downloading various citation formats such as EndNote XML, RIS, or BibTeX. This functionality has been requested in [#3140](https://github.com/IQSS/dataverse/issues/3140) and [#9994](https://github.com/IQSS/dataverse/issues/9994)
+
+### Files Endpoint Extended
+
+The API endpoint `api/files/{id}` has been extended to support the following optional query parameters:
+
+- **includeDeaccessioned**: Indicates whether or not to consider deaccessioned dataset versions in the latest file search. (Default: `false`).
+- **returnDatasetVersion**: Indicates whether or not to include the dataset version of the file in the response. (Default: `false`).
+
+A new endpoint `api/files/{id}/versions/{datasetVersionId}` has been created. This endpoint returns the file metadata present in the requested dataset version. To specify the dataset version, you can use `:latest-published`, `:latest`, `:draft` or `1.0` or any other available version identifier.
+
+The endpoint supports the *includeDeaccessioned* and *returnDatasetVersion* optional query parameters, as does the `api/files/{id}` endpoint.
+
+`api/files/{id}/draft` endpoint is no longer available in favor of the new endpoint `api/files/{id}/versions/{datasetVersionId}`, which can use the version identifier ``:draft`` (`api/files/{id}/versions/:draft`) to obtain the same result.
+
+### Datasets, Dataverse Collections, and Datafiles Endpoints Extended
+
+The API endpoints for getting datasets, Dataverse collections, and datafiles have been extended to support the following optional 'returnOwners' query parameter.
+
+Including the parameter and setting it to true will add a hierarchy showing which dataset and dataverse collection(s) the object is part of to the json object returned. 
+
+### Endpoint Fixed: Datasets Metadata
+
+The API endpoint `api/datasets/{id}/metadata` has been changed to default to the latest version of the dataset that the user has access.
+
+### Experimental Make Data Count processingState API
+
+An experimental Make Data Count processingState API has been added. For now it has been documented in the (developer guide)[https://guides.dataverse.org/en/6.2/developers/make-data-count.html#processing-archived-logs].
+
```

Comment:
At the end of the API section, should we say, "For a full list of backward-incompatible API changes, please see the "breaking changes" section below"?